src/main/java/calculator/exception/Exception.java

```diff
@@ -0,0 +1,36 @@
+package calculator.exception;
+
+import java.util.ArrayList;
+
+public class Exception {
+    public static int IsPositiveNum(String s){
+        try{
+            if(s == ""){
+                throw new NumberFormatException();
+            }
+            int num = Integer.parseInt(s);
+            if(num <= 0){
+                throw new IllegalArgumentException();
+            }
+            return num;
+        }
+        catch(NumberFormatException e){
+            throw new IllegalArgumentException();
+        }
+    }
+    public static void IsDelemeterChar(char c){
+        if(Character.isDigit(c)){
+            throw new IllegalArgumentException();
+        }
+    }
```

Comment:
`IllegalArgumentException()` 안에 String으로 예외 메시지를 지정할 수 있습니다!
예외 메시지가 있으면 원인 파악이 더 쉬워질 것 같아요 👍
추가로 Java 코딩 컨벤션이 전체적으로 맞지 않는 것으로 보입니다!
띄어쓰기 등 사소한 부분은, 인텥리제이 쓰신다면 Command + Option + L / Ctrl + Alt + L 로 코드 스타일 정리할 수 있어요!
```suggestion
    public static void isDelimeterChar(char c) {
        if (Character.isDigit(c)) {
            throw new IllegalArgumentException("~~~~일 수 없습니다.");
        }
    }
```